frontend/src/components/pages/student/StudentLoginPage.tsx

```diff
@@ -62,6 +71,15 @@ const StudentLoginPage = (): React.ReactElement => {
   const image = STUDENT_SIGNUP_IMAGE;
   const form = (
     <>
+      {loading && (
+        <Spinner
+          color="blue.300"
+          emptyColor="gray.200"
+          size="md"
+          speed="0.65s"
+          thickness="4px"
```

Comment:
Instead of adding all these props in both places (`LoadingState`), it might be nice to add these directly to the spinner `baseProps`/`default` variant in the theme? That way if we ever need to use a spinner in the future it will come pre-styled. Of course the `size` prop would still have to be set each time.

Reply:
good idea!

Reply:
oops just kidding. i think we actually can't set themes for certain props, i.e the ones that are not "stylistic": https://github.com/chakra-ui/chakra-ui/issues/1424

i'll make a custom component instead (:

Reply:
Rip oh well, sounds like a good solution!